source/consumers/elastic/lambda_handler.py

```diff
@@ -561,12 +561,37 @@ def connect_es(endpoint):
         return es_client
 
 
-# def delete_document(es_object, index, doc_id):
-#     es_index = "mie{index}".format(index=index).lower()
-#     es_object.delete(
-#         index=es_index,
-#         id=doc_id
-#     )
+def delete_asset_metadata_by_index(es_object, asset_id, index):
```

Comment:
Created this method in case we want to allow deletes of specific metadata types from the UI at some point